pre_commit_hooks/terradoc/fmt.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+terradoc fmt $1 -w
```

Comment:
Can we have a guard here to ensure terradoc is installed? eg https://github.com/mineiros-io/pre-commit-hooks/blob/master/pre_commit_hooks/markdown/markdown-link-check.sh#L5